ui/components/app/clear-metametrics-data/ClearMetaMetricsData.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useContext } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  hideDeleteMetaMetricsDataModal,
+  markingMetaMetricsDataDeletion,
+  continueRecordingMetaMetricsData,
+  stopRecordingMetaMetricsData,
+  openDataDeletionErrorModal,
+} from '../../../ducks/app/app';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import {
+  Box,
+  Button,
+  ButtonSize,
+  ButtonVariant,
+  Modal,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Text,
+} from '../../component-library';
+import {
+  AlignItems,
+  BlockSize,
+  Display,
+  FlexDirection,
+  JustifyContent,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import { createMetaMetricsDataDeletionTask } from '../../../store/actions';
```

Comment:
Perhaps this PR should stay in draft until the controller exists? As-is this is not mergable because this function does not exist.